main.go

```diff
@@ -127,6 +138,17 @@ func main() {
 		Scheme:    duros.GRPCS,
 		Log:       zap.NewRaw().Sugar(),
 	}
+
+	if apiEndpoint != "" && apiCA != "" && apiCert != "" && apiKey != "" {
```

Comment:
I think `Endpoints` also needs to be replaced because otherwise it still tries to connect with the wrong stuff:

```
❯ stern duros
+ duros-controller-6ddb87c66d-h68wk › duros-controller
duros-controller-6ddb87c66d-h68wk duros-controller I0706 09:46:03.623611       1 request.go:655] Throttling request took 1.046327379s, request: GET:https://10.48.0.1:443/apis/cert-manager.io/v1alpha3?timeout=32s
duros-controller-6ddb87c66d-h68wk duros-controller 2021-07-06T09:46:04.529Z     INFO    controller-runtime.metrics      metrics server is starting to listen    {"addr": ":8080"}
duros-controller-6ddb87c66d-h68wk duros-controller {"level":"info","ts":1625564764.5828211,"msg":"connecting...","client":"duros-go","targets":"10.131.44.1:443,10.131.44.2:443,10.131.44.3:443","client-id":"0z2zvju"}
duros-controller-6ddb87c66d-h68wk duros-controller {"level":"info","ts":1625564764.5830061,"msg":"connecting securely"}
```